src/commands/content.ts

```diff
@@ -11,8 +11,10 @@ export const content: CommandDefinition = {
     executor: (msg) => msg.channel.send(makeEmbed({
         title: 'FlyByWire Support | Checking your aircraft version',
         description: makeLines([
-            'In order to check which version of the A32NX you have installed, please open the MSFS Content Manager from the second page on the main menu and search for \'A32NX\'. '
+            'In order to check which version of the A32NX you have installed, please open the MSFS Content Manager from the second page on the main menu and search for \'flybywire\'. '
           + 'Then take a screenshot showing the results and send it here in [#Support](https://discord.gg/U2Askt6r) as shown below. ',
```

Comment:
Change discord link to vanity url

Reply:
Hadn't used vanity as going directly to support rather than inviting to server but link was incorrect and has been updated with a perm one.  Other option would be just to remove the link entirely and leave #support open to a bit of common sense. :)

![image](https://user-images.githubusercontent.com/87286435/133096622-a969e4ee-2dc8-43ec-a093-ba1573cdd239.png)

Reply:
It would be much safer just to invite the user to the server through the vanity url in the event of a invite purge for some reason